src/kOS/Module/kOSVesselModule.cs

```diff
@@ -19,6 +19,7 @@ public class kOSVesselModule : VesselModule
         private bool initialized = false;
         private Vessel parentVessel;
         private bool hasRemoteTech = false;
+        private int counterRemoteTechRefresh = 0;
```

Comment:
Okay I'll try it with 23 then.  (I've also moved the hardcoded `25` to a `private const int` identifier instead just for readability and possible fiddling later if the physics rate ever changes in future releases.